PasswordVerifier/src/com/company/PasswordVerifier.java

```diff
@@ -0,0 +1,81 @@
+public class PasswordVerifier {
+
+    public static void main(String[] args) {
+        String pass = "abdsdscdEF";
+        System.out.println("Password is : "+pass);
+        try {
+            System.out.println("Response is : "+verify(pass));
+        } catch (InvalidPasswordException e) {
+            System.out.println("Response is : "+e.getMessage());
+        }
+    }
+
+    public static String verify(String pass) throws InvalidPasswordException {
+        int count=0;
+        boolean isupper=false;
```

Comment:
For improved readability I'd suggest adopting a variable naming convention. Usually each language has an idiomatic one. Snake Case and Camel Case are two common ones.
Snake Case:
`is_upper`

Camel Case:
`isUpper`